massgov/openmass
# DP-31793 header alerts

**Description:**
The mass_alert module is modified for the icon change for "notice."

**Jira:** (Skip unless you are MA staff)
DP-31793


**To Test:**
Sample page:  /how-to/qag-request-help-with-a-computer-problem
The sample page only has an item for "alert."  Need to edit the item to "notice" or add another alert content with "notice."
- [ ] Find the component has a gray border.
- [ ] Find the border doesn't add up (= get thicker) when it has multiple alert items.
- [ ] The icon for the notice is charged to the bell and it's green.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
